### PR TITLE
feat: "Loading your neurons..." spinner

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

We want to voting feature to be available only when the neurons store has been fully initialized but we also want to provide a smooth UI. That's why instead of hiding the all proposal detail page while we load the information we need we can display the static content - i.e. the proposal detail - as fast as we can and display another spinner dedicated to the loading of the neurons.

Worth to note that the store is now loaded only once, therefore user should only face this spinner the first time the store is loaded.

# Changes

- split `ProposalDetail` checks for `proposalInfoStore` and `neuronsReady` to display two distinct spinner - i.e. display the `ProposalDetailCard` as fast as possible

# Screenshot

ezgif com-gif-maker (2)

### CODE REVIEW

File: frontend/svelte/src/routes/ProposalDetail.svelte [hunk 1]
```diff
@@ -110,17 +110,42 @@
...
         <Spinner />
       {/if}
     </section>
   </HeadlessLayout>
 {/if}
+
+<style lang="scss">
+  .loader {
+    display: flex;
+    flex-direction: column;
```
Comment (mstrasinskis) [inline frontend/svelte/src/routes/ProposalDetail.svelte:142]: Will it look w/o `flex-direction: column;`?

File: frontend/svelte/src/tests/routes/ProposalDetail.spec.ts [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+import en from "../mocks/i18n.mock";
+import { MockLedgerCanister } from "../mocks/ledger.canister.mock";
+import { buildMockNeuronsStoreSubscribe } from "../mocks/neurons.mock";
+import {
+  mockEmptyProposalsStoreSubscribe,
+  mockProposals,
+} from "../mocks/proposals.store.mock";
+import { mockRouteStoreSubscibe } from "../mocks/route.store.mock";
+
+describe("NeuronDetail", () => {
```
Comment (mstrasinskis) [inline frontend/svelte/src/tests/routes/ProposalDetail.spec.ts:23]: There is a typo in the word "Proposal" ;)

Comment (mstrasinskis) [inline frontend/svelte/src/tests/routes/ProposalDetail.spec.ts:23]: There is a typo in the word "Proposal" ;)

**Review by mstrasinskis — APPROVED** [review]
LGTM, just a minor comments.
Thanks for the test 👍

File: frontend/svelte/src/tests/routes/ProposalDetail.spec.ts (hunk 2 above)
Comment (peterpeterparker) [reply frontend/svelte/src/tests/routes/ProposalDetail.spec.ts]: These copy paste 😅

File: frontend/svelte/src/routes/ProposalDetail.svelte (hunk 1 above)
Comment (peterpeterparker) [reply frontend/svelte/src/routes/ProposalDetail.svelte:142]: Without `flex-direction: column;`

Capture d’écran 2022-04-20 à 06 52 10

Without `flex`

Capture d’écran 2022-04-20 à 06 51 28

**Pull request merged**